.storybook/webpack.config.js

```diff
@@ -6,31 +6,58 @@
 const path = require('path')
 const webpack = require('webpack')
 const fs = require('fs')
+const config = require('../build/commands/lib/config')
+const genTsConfig = require('../build/commands/lib/genTsConfig')
 const { fallback, provideNodeGlobals } = require('../components/webpack/polyfill')
+const generatePathMap = require('../components/webpack/path-map')
 
-const buildConfigs = ['Component', 'Static', 'Debug', 'Release']
-const extraArchitectures = ['arm64', 'x86']
+// Choose which brave-core build directory to look for pre-compiled
+// resource dependencies:
+// 1. Default for local builds for the actual platform / architecture
+// 2. platform / architecture overriden by environment variables
+// 3. most recently built - this caters to the common scenario when a
```

Comment:
I would personally not implement the "most recently built" logic. It is ambiguous and has already caused problems for me. Yes, that was with the `gen` subdirectory; But the principle remains the same and I strongly suspect that there will be a day where it causes a confusing surprise for another developer (directly or through CI).

Reply:
I'm keeping it to provide backwards compatibility. Since this is 99% a local dev debug and development preview tool, the risk is minimal.

Reply:
I have some concerns about this PR as well @mherrmann and discussing with @petemill